### PR TITLE
Update baibhav-tiwari.json

### PR DESCRIPTION
**Pull request by baibhav-tiwari**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4495"></a>

### CODE REVIEW

**Review by priyankarpal — DISMISSED** [review]
LGTM  👍🏻

**Review by AbbeyIT — DISMISSED** [review]
LGTM.

File: data/baibhav-tiwari.json [hunk 1]
```diff
@@ -3,8 +3,26 @@
...
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/BaibhavTiwari11"
+    },
+    {
+      "icon": "FaGithub",
+      "url": "https://github.com/baibhav-tiwari"
+    },
+    {
+      "icon": "SiHashnode",
```
Comment (kumarsonsoff3) [inline data/baibhav-tiwari.json:22]: You might want to add the LinkedIn icon here, which should be `FaLinkedin`. 

```suggestion
      "icon": "FaLinkedin",
```

File: data/baibhav-tiwari.json [hunk 2]
```diff
@@ -3,8 +3,26 @@
   "displayStatsPublic": true,
   "type": "personal",
   "bio": "**Web3** | **Web-developer** | **React** | **Open_Source**",
```
Comment (priyankarpal) [inline data/baibhav-tiwari.json:5]: Remove the underline it's not necessary
```suggestion
  "bio": "**Web3** | **Web-developer** | **React** | **Open Source**",
```

File: data/baibhav-tiwari.json [hunk 3]
```diff
@@ -18,45 +36,50 @@
...
+      "color": "red",
       "description": "It was my first programming language which I started Learning."
     },
     {
       "title": "Started Learning Web development",
       "date": "2022",
       "icon": "FaGlobe",
-      "color": "black",
+      "color": "green",
       "description": "i was always interested in creating websites, so i decided to learn web dev."
```
Comment (priyankarpal) [inline data/baibhav-tiwari.json:61]:
``` suggestion
      "description": "I was always interested in creating websites, so I decided to learn web dev."
```

File: data/baibhav-tiwari.json [hunk 4]
```diff
@@ -18,45 +36,50 @@
...
     },
     {
       "title": "Social Media",
       "date": "2022",
       "icon": "FaAndroid",
-      "color": "black",
+      "color": "green",
       "description": "Firstly i thought Twitter is boring but as i started using it more i started loving it."
```
Comment (priyankarpal) [inline data/baibhav-tiwari.json:68]:
``` suggestion
      "description": "Firstly I thought Twitter is boring but as I started using it more I started loving it."
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Just need some little bit changes

**Comment by baibhav-tiwari** [comment]
> Just need some little bit changes

Done as you suggested

**Review by baibhav-tiwari — COMMENTED** [review]
The older one is also the same thing isn't it?

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**